### PR TITLE
fix(docs): horizontal scroll in mobile main page

### PR DESCRIPTION
**Pull request by rinaok**
now:
Screenshot 2022-11-07 at 15 44 01

after fix:
Screenshot 2022-11-07 at 15 46 20

### CODE REVIEW

**Review by olaf-k — DISMISSED** [review]
LGTM (iPhone/iPad)

File: libs/components/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Only one theme is required to be loaded.
 
 ### Fonts (Prerequisite)
 
-- `node_modules/@vonage/vivid/styles/fonts/spezia.css` - Loads the *Spezia* variable font and defines its font face values. *Spezia* is Vonage's branded font and is required by most Vivid components. folder also contains the font files.
+- `node_modules/@vonage/vivid` `/styles/fonts/spezia.css` - Loads the *Spezia* variable font and defines its font face values. *Spezia* is Vonage's branded font and is required by most Vivid components. folder also contains the font files.
```
Comment (yinonov) [inline libs/components/README.md:52]: I don't get it

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/README.md (hunk 1 above)
Comment (yinonov) [reply libs/components/README.md:52]: ok I now get it. code tags don't break lines. something like `word-break: break-word;` applied to `code` would resolve the issue for good even if occurs in other docs.

question is, should we add it to the core styles package or the docs website

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/README.md (hunk 1 above)
Comment (rinaok) [reply libs/components/README.md]: I added it to the article.

**Review by yinonov — APPROVED** [review]

**Pull request merged**